app/array.json

```diff
@@ -0,0 +1 @@
+[{"type":"tEXt"},{"type":"select"},{"type":"button"},{"type":"form"},{"type":"label"},{"type":"email"}, {"type":"button"}]
```

Comment:
Requirements: Your logic is slightly flawed in that your example you are not accounting for the id or name fields of forms, which are a key points of how forms work in html.